docs/visualize/heatmap.asciidoc

```diff
@@ -0,0 +1,156 @@
+[[heatmap]]
+== Heat map
+
+Heat maps are graphical representations of data where the individual values are represented as colors.
+
+NOTE: Heat map has been replaced with <<maps>>, which offers more functionality and is easier to use.
+
+[float]
+[[heatmap-aggregation]]
+=== Supported aggregations
+
+Heat maps support metric, sibling pipeline, bucket, and parent pipeline aggregations.
+
+[float]
+[[heatmap-metric-aggregation]]
+==== Metric aggregations
+
+The color for each matrix position is determined by the metric aggregation.
+
+The following metric aggregations are supported:
+
+{ref}/search-aggregations-metrics-valuecount-aggregation.html[Count]:: Returns a raw count of
+the elements in the index pattern.
+
+{ref}/search-aggregations-metrics-avg-aggregation.html[Average]:: Returns the average of a numeric
+field.
+
+{ref}/search-aggregations-metrics-sum-aggregation.html[Sum]:: Returns the total sum of a numeric
+field.
+
+{ref}/search-aggregations-metrics-min-aggregation.html[Min]:: Returns the minimum value of a
+numeric field.
+
+{ref}/search-aggregations-metrics-max-aggregation.html[Max]:: Returns the maximum value of a
+numeric field.
+
+{ref}/search-aggregations-metrics-cardinality-aggregation.html[Unique Count]:: Returns
```

Comment:
Should Unique count, Top hit, and other aggregations with two words be sentence case?

Reply:
I can change Unique count, but the others are Elasticsearch terms.